### PR TITLE
Add documentation link at docs.elpis.cloud

### PR DESCRIPTION
**Pull request by harrykeightley**

### CODE REVIEW

**Review by nicklambourne — APPROVED** [review]
Looks fine, is there a tf plan to go with these changes? Good practice to include in PRs if its not part of the tooling already.

File: architecture/environments/prod/main.tf [hunk 1]
```diff
@@ -21,6 +21,15 @@ module "zones" {
   root_zone_url  = local.website_url
 }
 
+# Documentation
+resource "google_dns_record_set" "documentation" {
+  name         = "docs.${module.zones.root_zone.dns_name}"
+  managed_zone = module.zones.root_zone.name
+  type         = "CNAME"
+  ttl          = 300
+  rrdatas      = ["elpiscloud.readthedocs.io."]
```
Comment (aviraljain99) [inline architecture/environments/prod/main.tf:30]: I am not sure about this but is there supposed to be a '.' at the end as well? Or just elpiscloud.readthedocs.io?

Comment (aviraljain99) [reply architecture/environments/prod/main.tf:30]: Also, this is just me not understanding all this very well but you have set it up such that elpiscloud.readthedocs.io maps to docs.elpis.cloud - but then where is docs.elpis.cloud given an IP address?

Comment (nicklambourne) [reply architecture/environments/prod/main.tf:30]: All of the examples here: https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/dns_record_set have the trailing period. I guess we'll find out on plan/apply.

Comment (nicklambourne) [reply architecture/environments/prod/main.tf:30]: > where is docs.elpis.cloud given an IP address

This entry is just adding a DNS record, the actual IP allocated where the e.g. EC2 instances are created and resolved (I'm assuming) by reference in this line `name = docs.${module.zones.root_zone.dns_name}`.

Comment (nicklambourne) [reply architecture/environments/prod/main.tf:30]: It's also a CNAME record, which means it's just pointing at a URL, not an IP address.

**Comment by harrykeightley** [comment]
> Looks fine, is there a tf plan to go with these changes? Good practice to include in PRs if its not part of the tooling already.

The TF Plan is made as part of the cloudbuild-on-push check that has to pass before any pull requests can be merged.

File: architecture/environments/prod/main.tf (hunk 1 above)
Comment (benfoley) [reply architecture/environments/prod/main.tf:30]: The dot indicates that the domain name is complete. A domain without a dot may get the resolver's default domain attached to it.

**Comment by nicklambourne** [comment]
> > Looks fine, is there a tf plan to go with these changes? Good practice to include in PRs if its not part of the tooling already.
> 
> The TF Plan is made as part of the cloudbuild-on-push check that has to pass before any pull requests can be merged.

And where do we go to see that output?

**Review by aviraljain99 — APPROVED** [review]

**Comment by harrykeightley** [comment]
> > > Looks fine, is there a tf plan to go with these changes? Good practice to include in PRs if its not part of the tooling already.
> 
> > 
> 
> > The TF Plan is made as part of the cloudbuild-on-push check that has to pass before any pull requests can be merged.
> 
> 
> 
> And where do we go to see that output?

In the checks section above/on any pull request

**Pull request merged**